week08/69. Sqrt(x).cpp

```diff
@@ -0,0 +1,17 @@
+//TC: O(x)
```

Comment:
What?!

You did a binary search but wrote the complexity as linear!!!!!!!!!! Man!

Reply:
//TC: O()
//MC: O()
ei format ta ami aage upore likhe then code kori. Pore folder create kore sob eksathe copy paste kori. Ejonno emon vulgulo hoye geche ( ar ekhane O(logx) na likhe O(x) likhchi).